src/js/index/todolist.js

```diff
@@ -0,0 +1,121 @@
+const $todoInput = document.querySelector("#new-todo-title");
+const $todoList = document.querySelector("#todo-list");
+const $todoCount = document.querySelector("strong");
+const $filters = document.querySelector(".filters");
+
+$todoInput.addEventListener("keyup", onAddTodoItem);
+$todoList.addEventListener("click", onToggleTodoItem);
+$todoList.addEventListener("dblclick", onEditTodoItem);
+$filters.addEventListener("click", onFilter);
+
+function onAddTodoItem(event) {
+    const todoTitle = event.target.value;
+    const todoList = document.getElementById("todo-list");
+    if (event.key === "Enter" && todoTitle !== "") {
+        todoList.insertAdjacentHTML("beforeend", renderTodoItemTemplate(todoTitle));
+        updateTodoCount();
+        event.target.value = "";
+        changeState();
+    }
+}
+
+function onToggleTodoItem(event) {
+    if (event.target.classList.contains("toggle")) {
+        event.target.closest("li").classList.toggle("completed");
+        event.target.closest("input").toggleAttribute("checked");
+        changeState()
+    } else if (event.target.classList.contains("destroy")) {
+        event.target.closest("li").remove();
+        updateTodoCount();
+        changeState();
+    }
+}
+
+function onEditTodoItem(event) {
+    if (event.target.classList.contains("label")) {
+        const originalValue = event.target.innerText;
+        event.target.closest("li").classList.add("editing");
+        event.target.parentElement.parentElement.querySelector(".edit").focus();
+        event.target.closest("li")
+            .addEventListener("keyup", function (event) {editTodoItem(event, originalValue)});
+    }
+}
+
+function editTodoItem(event, originalValue) {
+    switch (event.key) {
+        case "Enter" :
+            return updateTodoItem(event)
```

Comment:
```suggestion
            updateTodoItem(event);
            return;
```

updateTodoItem 이 무언가를 리턴하는 줄 알았는데, 아무 값도 리턴하지 않네요!

그렇다면은 이렇게 두 줄로 써주는게 좋은 것 같습니다 :smiley: